distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc

```diff
@@ -92,57 +92,57 @@ Because ${branding} is built on top of Apache Karaf, ${branding} can use the Kar
 .Within the ${branding} console
 ----
 ${branding-lowercase}${at-symbol}local> feature:install wrapper
-${branding-lowercase}${at-symbol}local> wrapper:install -s AUTO_START -n ${ddf-branding-lowercase} -d ${ddf-branding-lowercase} -D "${branding} Service"
+${branding-lowercase}${at-symbol}local> wrapper:install -s AUTO_START -n ${branding-lowercase} -d ${branding-lowercase} -D "${branding} Service"
 ----
 +
-. (Windows users skip to next step) (All *NIX) If ${branding} was installed to run as a non-root user (recommended,) edit `INSTALLATION_HOME/bin/${branding-lowercase}-service`.
+. (Windows users skip to next step) (All *NIX) If ${branding} was installed to run as a non-root user (recommended,) edit `<${branding}_HOME>/bin/${branding-lowercase}-service`.
 +
 Change:
 +
-.INSTALLATION_HOME/bin/${branding-lowercase}-service
+.<${branding}_HOME>/bin/${branding-lowercase}-service
 ----
 #RUN_AS_USER=
 ----
 +
 to:
 +
-.INSTALLATION_HOME/bin/${branding-lowercase}-service
+.<${branding}_HOME>/bin/${branding-lowercase}-service
 ----
 RUN_AS_USER=<${branding-lowercase}-user>
 ----
 +
 . Set the memory in the wrapper config to match with ${branding} default memory setting.
 +
-.INSTALLATION_HOME/etc/${branding-lowercase}-wrapper.conf
+.<${branding}_HOME>/etc/${branding-lowercase}-wrapper.conf
 [source,java,linenums]
 ----
 #Add the following:
-wrapper.java.additional.10=-D${ddf-branding-lowercase}.home=%KARAF_HOME%
-wrapper.java.additional.12=-Dderby.storage.fileSyncTransactionLog=true
-wrapper.java.additional.11=-server
+wrapper.java.additional.10=-D${branding-lowercase}.home=%KARAF_HOME%
```

Comment:
I think this is supposed to be `ddf.home` regardless of the distribution you're running.